tract_querier/tract_math/operations.py

```diff
@@ -5,84 +5,324 @@
 except ImportError:  # Python 2.6 fix
     from ordereddict import OrderedDict
 
-import numpy
-from numpy import linalg
-
 import nibabel
 from nibabel.spatialimages import SpatialImage
 
 from ..tractography import Tractography, tractography_to_file, tractography_from_files
 
+import os
+import sys
+import traceback
+from collections import OrderedDict  #Only python 2.7
+import numpy
 
-@tract_math_operation(': counts the number of tracts', needs_one_tract=False)
-def count(tractographies):
-    results = {'tract file #': [], 'number of tracts': []}
-    for i, tractography in enumerate(tractographies):
-        results['tract file #'].append(i)
-        results['number of tracts'].append(len(tractography.tracts()))
+def _set_dictionary_from_use_file_names_as_index(optional_flags,
+                                         tractographyName,default_tractographyName,
```

Comment:
I don't see the need for a function name starting with the underscore here. We could just eliminate it